ote/src/cljs/ote/views/user_edit.cljs

```diff
@@ -0,0 +1,84 @@
+(ns ote.views.user-edit
```

Comment:
Should this be only for admins? Users have their own page for editing their data

Reply:
Yes only for admins